### PR TITLE
Text to speech / Story reader created.

### PR DESCRIPTION
**Pull request by simarpreetsingh-019**
# Description  

Text to speech / Story Reader .py created.
Solved #355  

# Type of change
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code
- [x]  I have created easy to understand README.md

### CODE REVIEW

**Comment by simarpreetsingh-019** [comment]
Error due to having no newline at the end removed. It is ready to merge.

File: Story Reader/main.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from time import sleep
+
+import pyttsx3 #pip install pyttsx3
+import PyPDF2 #pip install PyPDF2
+
+story = open('mybook.pdf', 'rb') #Name of your script make sure it's in pdf format
```
Comment (HarshCasper) [inline Story Reader/main.py:6]: Take the File Input from User. Don't hardcode it

File: Story Reader/Readme.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+<div align="center">
+<img src="audiobook.png" height=500 width="500">
+</div>
+
+# Audiobook 
+
+Read any audio book from pdf using python.
+
+### Modules used:
```
Comment (HarshCasper) [inline Story Reader/Readme.md:11]: Format this section of README properly

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Story Reader/main.py (hunk 1 above)
Comment (simarpreetsingh-019) [reply Story Reader/main.py]: issue solved.  please review

File: Story Reader/Readme.md (hunk 2 above)
Comment (simarpreetsingh-019) [reply Story Reader/Readme.md:10]: done

**Pull request merged**